config.lua

```diff
@@ -819,8 +827,138 @@ Config.Doors = {
+        Difficulty = 3,
+        Alert = true,
+    },
+    [4248740658]  = {
+        Pos = vector3(346.23724365234375, 1468.9339599609375, 178.72691345214844),
+        Name = "Cell Door",
+        DoorState = 1,
+        Permissions = Config.Permissions.FortWallace,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [1223148325]  = {
+        Pos = vector3(363.3249206542969, 1488.7296142578125, 179.66148376464844),
+        Name = "Armory Door",
+        DoorState = 1,
+        Permissions = Config.Permissions.FortWallace,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [2977782592]  = {
+        Pos = vector3(359.58148193359375, 1510.486328125, 179.16412353515625),
+        Name = "Stable Back Door",
+        DoorState = 1,
+        Permissions = Config.Permissions.FortWallace,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+	
+	-- Sisika Doors
+	-- Doors ID
+    [3158090902]  = {
+        Pos = vector3(3349.908935546875, -645.2470703125, 44.35463714599609),
+        Name = "Front Gate 1",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [2381465602]  = {
+        Pos = vector3(3350.638671875, -647.97021484375, 44.35463714599609),
+        Name = "Front Gate 2",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [2617210026]  = {
+        Pos = vector3(3325.8056640625, -705.5444946289062, 43.38306045532226),
+        Name = "Back Gate 1 (Left)",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [3093514944]  = {
+        Pos = vector3(3327.572265625, -707.517822265625, 43.38306045532226),
+        Name = "Back Gate 2 (Right)",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [1121239638]  = {
+        Pos = vector3(3331.8154296875, -700.0858764648438, 43.06853103637695),
+        Name = "Back Gate 3 (Left)",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    [906662604]  = {
+        Pos = vector3(3333.580810546875, -702.0559692382812, 43.06853103637695),
+        Name = "Back Gate 4 (Right)",
+        DoorState = 1,
+        Permissions = Config.Permissions.Sisika,
+        BreakAble = Config.Lockpicks.location,
+        Difficulty = 3,
+        Alert = true,
+    },
+    -- []  = { -- Door Hash Code Not Available
```

Comment:
this needs to be removed if theres no ids.

Reply:
> this needs to be removed if theres no ids.

Fix